modules/ROOT/pages/release-notes.adoc

```diff
@@ -171,9 +171,45 @@ Bonita Runtime produces a warning log if the filter is set and ignores it.
 
 The tenant creation method has been deprecated following our 2021.1 decision to deprecate the Multi-Tenants architecture.
 
+=== Potential need of declaring external Maven Repository
+
+From version 2022.1-u2, if you embed *bonita-server(-sp).jar* in your application, you need to declare additional Maven Repository in your pom.xml / build.gradle. This is due to Security Fixes that required the use of a dependency NOT present in default Maven Repository (aka Maven Central). To do so, add the following code:
+
+==== Pom.xml
+
+```xml
+<repositories>
+    [...]
+    <repository>
+        <id>terracotta.org</id>
+        <name>terracotta.org</name>
+        <url>https://repo.terracotta.org/maven2/</url>
+    </repository>
+</repositories>
+```
+
+==== build.gradle
+
+[source,groovy]
+----
+repositories {
+    mavenCentral()
+    maven {
+        url 'https://repo.terracotta.org/maven2/'
+    }
+}
+----
+
+Of course, you can abstract this additional complexity by putting this configuration in your https://maven.apache.org/repository-management.html[enterprise repository manager] which acts a proxy to any external repositories you may have.
```

Comment:
This applies to both maven and gradle so putting it before the example will make the message more visible.
The repository manager usage is not only for simplification, but if the Runtime Team agrees with the current wording, it's fine to me.

Reply:
done

Reply:
thanks